DepositeCalcTests/Pages/CalculatorPage.cs

```diff
@@ -92,10 +95,19 @@ public string StartDateYear
             get => new SelectElement(YearDropDown).SelectedOption.Text;
             set => new SelectElement(YearDropDown).SelectByText(value);
         }
+
         public string StartDateDay
         {
             get => new SelectElement(DayDropDown).SelectedOption.Text;
             set => new SelectElement(DayDropDown).SelectByText(value);
         }
+
+        public void ClickOnSettingsLnk()
+        {
+            SettingsLnk.Click();
+        }
```

Comment:
1. Change name of the method;
2. It is good practice to return instance of the result page.
```suggestion
        public SettingsPage OpenSettings()
        {
            SettingsLnk.Click();
            return new SettingsPage(driver);
        }
```